docs/api-guide/authentication.md

```diff
@@ -217,38 +210,6 @@ It is also possible to create Tokens manually through admin interface. In case y
     TokenAdmin.raw_id_fields = ('user',)
 
 
-#### Schema migrations
```

Comment:
Worked this around and added a note in the settings paragraph about DRF using migrations.